packages/wallet-extensions/package.json

```diff
@@ -29,7 +29,7 @@
       "bun": "./src/index.ts",
       "default": "./dist/src/index.js",
       "require": "./dist/src/index.cjs",
-      "types": "./dist/types/index.d.ts"
```

Comment:
That's a wrong way of doing it - `index.d.ts` is holding whole types for package - it's up to tsconfig.json to resolve that properly.